enforcer_interface.go

```diff
@@ -23,6 +23,8 @@ import (
 )
 
 var _ IEnforcer = &Enforcer{}
+var _ IEnforcer = &SyncedEnforcer{}
+var _ IEnforcer = &CachedEnforcer{}
```

Comment:
I'm not sure if correct. @hsluoyz

Reply:
This is correct, see: https://stackoverflow.com/questions/10498547/ensure-a-type-implements-an-interface-at-compile-time-in-go